packages/block-library/src/accordion-content/edit.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	useBlockProps,
+	useInnerBlocksProps,
+	__experimentalUseBorderProps as useBorderProps,
+	__experimentalUseColorProps as useColorProps,
+	__experimentalGetSpacingClassesAndStyles as useSpacingProps,
+	__experimentalGetShadowClassesAndStyles as useShadowProps,
+} from '@wordpress/block-editor';
+/**
+ * External dependencies
+ */
+import clsx from 'clsx';
+
+export default function Edit( { attributes } ) {
+	const { allowedBlocks, templateLock, openByDefault, isSelected } =
+		attributes;
+	const borderProps = useBorderProps( attributes );
+	const colorProps = useColorProps( attributes );
+	const spacingProps = useSpacingProps( attributes );
+	const shadowProps = useShadowProps( attributes );
+
+	const blockProps = useBlockProps();
+	const innerBlocksProps = useInnerBlocksProps(
+		{
+			className: 'accordion-content__wrapper',
+			style: {
+				...spacingProps.style,
+			},
+		},
+		{
+			allowedBlocks,
+			template: [ [ 'core/paragraph', {} ] ],
+			templateLock,
+		}
+	);
+
+	return (
+		<div
+			{ ...blockProps }
+			className={ clsx(
+				blockProps.className,
+				colorProps.className,
+				borderProps.className,
+				{
+					[ `has-custom-font-size` ]: blockProps?.style?.fontSize,
+				}
+			) }
+			style={ {
+				...borderProps.style,
+				...colorProps.style,
+				...shadowProps.style,
+			} }
+			aria-hidden={ ! isSelected && ! openByDefault }
```

Comment:
Using `aria-hidden="true"` on concealed content will run into accessibility problems if the contained content has focusable content. I assume this is used to avoid `display: none`, since that causes problems with animating the state change, but it will also introduce accessibility problems. 

Might work if you find all focusable elements inside the content container and disable their keyboard navigability when aria-hidden is applied.

Reply:
Yes that's exactly right. I will try your suggestion of disabling keyboard navigability of focusable elements when aria-hidden is applied!

Reply:
> find all focusable elements inside the content container and disable their keyboard navigability when aria-hidden is applied.

@joedolson I understand this in the context of the front-end experience, but I'm wondering how it should be handled in the editor. The aria-hidden property is removed if the accordion item or any of its children are selected, which makes it so the contained content is always effectively keyboard navigable: 

https://github.com/user-attachments/assets/c88ac4b8-88e0-444d-9659-d6dd5a3cf7ec

Reply:
I addressed this in the front-end view: https://github.com/WordPress/gutenberg/pull/64119/commits/b522ddad5840bde81ae40be2a754d53a02eab518

@luisherranz I first tried to implement this on the server by by setting `data-wp-bind--tabindex` to the derived state, but because of how many blocks (like the file or navigation block) are rendered, the tabindex was being overridden, thus still ended up in the tab order. Curious if you have any thoughts or if what I've done above is a decent solution.

Reply:
If there's no keyboard path to reach the hidden content that doesn't cause it to become visible, then it's probably not necessary to also hide it in the editor. That should probably get checked over with a few different screen readers, however; I think that a screen reader user navigating with the virtual cursor would still be able to get to those items.

Reply:
So the thought is that we can remove the toggling of `aria-hidden` in the editor, since there is no keyboard path to reach the hidden content that doesn't cause it to become visible?

Reply:
Using `inert` is a good call; I'm not in the habit of suggesting it, since it's so newly available.

Reply:
Nice suggestion, using `insert` works as expected — child elements are not keyboard navigable when the accordion item is collapsed. Applied this change in [aebf612](https://github.com/WordPress/gutenberg/pull/64119/commits/aebf6125d9e9ae84239141bf5dca2c694c256364).